e2e/app/authenticated-page.ts

```diff
@@ -179,7 +179,7 @@ export default abstract class AuthenticatedPage extends BasePage {
 
   private async isSideNavDropdownOpen(): Promise<boolean> {
```

Comment:
Could you please add a description, like "the side Nav for this page is either docked to the right or open. We can determine the open state by looking for a visible Home icon" Or something. It's not obvious to someone who doesn't watch the UI closely.

Additionally, it's confusing that the method is named after the  side nav dropdown, but the implementation is in terms of a  home icon. Can we add a test data tag to the nav and check for it more directly?

Reply:
sorry. your feedback is awesome! I have comments for most "big" methods but don't do for all methods because I think some methods are easy to know what it suppose to do. For this method, I added what you said as comment. 

It is the easiest way to determine the open or close state by looking for visible `Home` icon. If someone is going to implement a test friendly tag in future, I'd be happy to update this code.